src/server/scripts/Commands/cs_reset.cpp

```diff
@@ -296,6 +314,447 @@ class reset_commandscript : public CommandScript
 
         return true;
     }
+
+    static bool HandleResetItemsEquipedCommand(ChatHandler* handler, Optional<PlayerIdentifier> target)
```

Comment:
Why is the playeridentifier an Optional if it is always required?

Reply:
@Nyeriah . It is never mandatory.
It takes the player name but if not provided it tries to get selected player.
AFter some tests getting the selected player  using optionnal target was making weird things because it was set to "nullopt".

So 2 uses cases possible : 

- You provide player name and command is executed with the value passed tu the command
- You do not provide the optionnal parameter then it tries to get selected player through chathandler.

It is not using target or self to ensure you acknowledge selecting your target because this command is destructuive. It avoids messing your own char ;)

Reply:
@Nyeriah Do you think it could be merged ?
I am planing to also add a .reset mail command